src/muya/lib/i18n/index.js

```diff
@@ -0,0 +1,3 @@
+import i18n from '../../../locales'
```

Comment:
I'm unsure about this because Muya is a pure JS library that shouldn't include `vue` too.

Reply:
Vue is just served as Vue-i18n's runtime. No other usages.